src/blocks/author-profile/components/inspector/StylePanel.js

```
@@ -0,0 +1,649 @@
+		// More content settings
+		showMoreContent,
+		moreContentBorderColor,
+		moreContentPadding,
+	} = attributes;
+
+	return (
+		<>
+			<PanelBody title={__('Block Background', 'author-profile-blocks')}>
+				<BaseControl
+					label={__('Background Color', 'author-profile-blocks')}
+				>
```

ESLint

[error] When using BaseControl component if a label property is passed an id property should also be passed.

(@wordpress/no-base-control-with-label-without-id)